publish/stage-maven-release.sh

```diff
@@ -149,3 +149,55 @@ mvn --settings="${mvn_settings}" \
 echo "==========================================="
 echo "Done."
 echo "==========================================="
+
+## Below commands will be executed if the user checks AUTO_RELEASE_STAGING_TO_PROD parameter in the
+## maven-sign-release job. See https://github.com/sonatype/nexus-maven-plugins/blob/main/staging/maven-plugin/README.md
+## for command reference.
+if [ "$AUTO_RELEASE" = true ] ; then
+    export MAVEN_OPTS=--add-opens=java.base/java.util=ALL-UNNAMED ## See https://issues.sonatype.org/browse/NEXUS-31214
+
+    echo "==========================================="
+    echo "Closing Staging Repository ${staging_repo_id}."
+    echo "==========================================="
+
+    mvn --settings="${mvn_settings}" \
+      org.sonatype.plugins:nexus-staging-maven-plugin:1.6.13:rc-close \
+      -DnexusUrl="${REPO_URL}" \
+      -DserverId=nexus \
+      -DautoReleaseAfterClose=true \
+      -DstagingProfileId="${STAGING_PROFILE_ID}" \
+      -DstagingRepositoryId="${staging_repo_id}"
+
+    echo "==========================================="
+    echo "Done."
+    echo "==========================================="
+
+    echo "==========================================="
+    echo "Release Staging Repository ${staging_repo_id}."
+    echo "==========================================="
+
+    mvn --settings="${mvn_settings}" \
+      org.sonatype.plugins:nexus-staging-maven-plugin:1.6.13:rc-release \
+      -DnexusUrl="${REPO_URL}" \
+      -DserverId=nexus \
+      -DstagingProfileId="${STAGING_PROFILE_ID}" \
+      -DstagingRepositoryId="${staging_repo_id}"
+
+    echo "==========================================="
+    echo "Done."
+    echo "==========================================="
+
+    echo "==========================================="
+    echo "Dropping Staging Repository ${staging_repo_id}."
```

Comment:
I remember on GUI there is an option to autodrop after release.
Is that option not in here?
That you have to manually drop?

Reply:
Yes, that flag is not working as expected with any of the `rc-*` commands.

Reply:
ok thanks.